pyomo/core/tests/unit/test_visitor.py

```diff
@@ -106,7 +106,7 @@ def test_identify_vars_vars(self):
         self.assertEqual( list(identify_variables(m.a**m.b[1])),
                           [ m.a, m.b[1] ] )
         self.assertEqual( list(identify_variables(m.a**m.b[1] + m.b[2])),
-                          [ m.a, m.b[1], m.b[2] ] )
+                          [ m.b[2], m.a, m.b[1]] )
```

Comment:
Why did this test baseline change?

Reply:
from @michaelbynum:
> If you look closely, all of the variables have the same value… The original test was incorrect, and the bool(==) behavior hid it.
Great!  It is very good that these tests are more robustly testing the result.